modules/radio/react/lib/Radio.tsx

```diff
@@ -44,6 +44,9 @@ const RadioContainer = styled('div')({
 const RadioInputWrapper = styled('div')<Pick<RadioProps, 'disabled'>>(
   {
     height: radioHeight,
+    width: radioWidth,
+    marginTop: '7px',
```

Comment:
How come this is 7px and checkbox is 3px?

Reply:
It was because they must have a min height of 32px in RadioGroup, but it does make sense to have them have the same height than `Checkbox` and `Switch` when they're standalone. I've updated `Radio` and moved the spacing in the context of a radio group in `RadioGroup` itself.